components/Grids/Grids.css

```diff
@@ -0,0 +1,24 @@
+.two-box-grid{
+    grid-template-columns: 1fr 1fr;
+}
+.three-box-grid{
+    grid-template-columns: 1fr 1fr 1fr;
+}
```

Comment:
Approving the changes.
Just a suggestion: class names can be improved, you may have a look at tailwind grid classnames.